File: MISSJENNY/age.c
#include <stdio.h>

int main() {
    int age;
    printf("Enter value of age: "); 
    scanf("%d", &age); 

    if (age >= 65) {
        printf("%d Age is greater than or equal to 65", age);
    } else {
        printf("%d age is less than 65", age);
    }

    return 0;
}

File: MISSJENNY/Age.java
import java.util.Scanner;

public class Age {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter The Value Of Age");
        int age = scanner.nextInt();

        String result = (age >= 65) ? "Age is greater than or equal to 65" : "Age is less than 65";
        System.out.println(result);
    }
}
